cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -116,6 +116,14 @@ def _read_h5ad(self, h5ad_path: Union[str, bytes, os.PathLike]):
 
         self.h5ad_path = h5ad_path
 
+    def _validate_encoding_version(self):
+        import h5py
+        with h5py.File(self.h5ad_path, "r") as f:
+            encoding_dict = dict(f.attrs)
+            encoding_version = encoding_dict["encoding-version"]
```

Comment:
use encoding_dict.get('encoding-version') to avoid KeyErrors for versions < 0.8.0